backend/src/utils/constants.ts

```diff
@@ -51,7 +51,7 @@ export const blankDashboardCR: DashboardConfig = {
       disableModelServing: false,
       disableProjectSharing: false,
       disableCustomServingRuntimes: false,
-      disableBiasMetrics: false,
+      disableTrustyAI: false,
```

Comment:
Wouldn't this fail when we implement drift? Would anyone want only bias and not drift? or vice-versa